### PR TITLE
Move `@itwin/presentation-opentelemetry` package from `itwinjs-core`

### PR DESCRIPTION
**Pull request by grigasp**
- Move `@itwin/presentation-opentelemetry` package from `itwinjs-core`.
- Fix internal API usages (closes https://github.com/iTwin/presentation/issues/100).
- Add an example and code snippets to our test app.

Related PR in `itwinjs-core` repo: https://github.com/iTwin/itwinjs-core/pull/5401

### CODE REVIEW

File: packages/opentelemetry/package.json [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  "name": "@itwin/presentation-opentelemetry",
+  "version": "4.0.0-dev.76",
+  "description": "Utilities to enable OpenTelemetry tracing of Presentation requests",
+  "license": "MIT",
+  "engines": {
+    "node": "^18.0.0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/iTwin/itwinjs-core.git",
```
Comment (saskliutas) [inline packages/opentelemetry/package.json:11]: Need to update these to point to the new repo

File: packages/opentelemetry/package.json [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    "name": "Bentley Systems, Inc.",
+    "url": "http://www.bentley.com"
+  },
+  "main": "lib/cjs/presentation-opentelemetry.js",
+  "typings": "lib/cjs/presentation-opentelemetry",
+  "scripts": {
+    "build": "npm run -s build:cjs",
+    "build:cjs": "tsc -p tsconfig.cjs.json",
+    "build:watch": "npm run -s build:cjs -- -w",
+    "clean": "rimraf lib .rush/temp/package-deps*.json",
```
Comment (saskliutas) [inline packages/opentelemetry/package.json:32]:
```suggestion
    "clean": "rimraf lib",
```

File: packages/opentelemetry/package.json [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    "cover": "nyc npm -s test",
+    "docs": "npm run -s docs:reference && npm run -s extract && npm run -s docs:changelog",
+    "docs:changelog": "cpx ./CHANGELOG.md ../../generated-docs/presentation/presentation-opentelemetry",
```
Comment (saskliutas) [inline packages/opentelemetry/package.json:35]:
```suggestion
    "docs:changelog": "cpx ./CHANGELOG.md ../../build/docs/reference/presentation-opentelemetry",
```

File: packages/opentelemetry/package.json [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    "docs:reference": "betools docs --includes=../../generated-docs/extract --json=../../generated-docs/presentation/presentation-opentelemetry/file.json --tsIndexFile=presentation-opentelemetry.ts --onlyJson",
```
Comment (saskliutas) [inline packages/opentelemetry/package.json:36]:
```suggestion
    "docs:reference": "betools docs --includes=../../build/docs/extract --json=../../build/docs/reference/presentation-opentelemetry/file.json --tsIndexFile=presentation-opentelemetry.ts --onlyJson",
```

File: packages/opentelemetry/package.json [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+    "extract": "betools extract --fileExt=ts --extractFrom=./src/test --recursive --out=../../generated-docs/extract",
```
Comment (saskliutas) [inline packages/opentelemetry/package.json:37]:
```suggestion
    "extract": "betools extract --fileExt=ts --extractFrom=./src/test --recursive --out=../../build/docs/extract",
```

File: change/gytis-drop-Node-12-14_2023-02-01-15-55.json [hunk 6]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (saskliutas) [inline change/gytis-drop-Node-12-14_2023-02-01-15-55.json:1]: These change files have different file name format. Can you make sure that `beachball` will handle them correctly?

**Review by saskliutas — CHANGES_REQUESTED** [review]

File: packages/opentelemetry/package.json (hunk 1 above)
Comment (grigasp) [reply packages/opentelemetry/package.json]: fixed

File: packages/opentelemetry/package.json (hunk 2 above)
Comment (grigasp) [reply packages/opentelemetry/package.json]: fixed

File: packages/opentelemetry/package.json (hunk 3 above)
Comment (grigasp) [reply packages/opentelemetry/package.json]: fixed

File: packages/opentelemetry/package.json (hunk 4 above)
Comment (grigasp) [reply packages/opentelemetry/package.json]: fixed

File: packages/opentelemetry/package.json (hunk 5 above)
Comment (grigasp) [reply packages/opentelemetry/package.json]: fixed

File: change/gytis-drop-Node-12-14_2023-02-01-15-55.json (hunk 6 above)
Comment (grigasp) [reply change/gytis-drop-Node-12-14_2023-02-01-15-55.json:1]:
```
beachball check
Checking for changes against "origin/master"
fetching latest from remotes "origin/master"
Your local repository already has change files for these packages:
  @itwin/presentation-opentelemetry
  @itwin/presentation-testing
Validating no private package among package dependencies
 OK!

No change files are needed
```

**Pull request merged**